analysis_runner/dataproc.py

```diff
@@ -210,6 +208,7 @@ def _add_start_job(  # pylint: disable=too-many-arguments
 
     # Note that the options and their values must be separated by an equal sign.
     # Using a space will break some options like --label
+    namespace = 'test' if ACCESS_LEVEL == 'test' else 'main'
```

Comment:
It's a simple check, but we duplicate this in a number of places (e.g. also `server/util.py` below). Is it worth putting this into a tiny shared function (either in the analysis-runner or `cpg-utils`)?

Reply:
+1 to this!!

Reply:
It actually removed something like that in the last cpg-utils PR 😅 But having a `Namespace` enum was an overkill anyway, agree that a simple function would be better